src/Data/DataIndexResponse.ts

```diff
@@ -0,0 +1,7 @@
+import { RawItem } from './RawItem'
+
+export interface DataIndexResponse {
+  results: RawItem[]
+  count?: number
+  continuation?: string
+}
```

Comment:
* [ ] FYI As long as there are no plans to add non-type exports to things like `DataIndexResponse` and `RawItem` a `Data/types.ts` could make more sense.